api_v3/lib/KalturaErrors.php

```diff
@@ -655,6 +655,7 @@ class KalturaErrors extends APIErrors
 	const MUST_FILTER_ON_ENTRY_OR_USER = 'MUST_FILTER_ON_ENTRY_OR_USER;;Must filter on entry ID or user ID';
 	const USER_ENTRY_FILTER_FORBIDDEN_FIELDS_USED = 'USER_ENTRY_FILTER_FORBIDDEN_FIELDS_USED;;UserEntry filter object forbidden fields used';
 	const USER_ENTRY_ALREADY_EXISTS = 'USER_ENTRY_ALREADY_EXISTS;;UserEntry for this type already exists';
+	const USER_ENTRY_FILTER_MISSSING_MANDATORY_PARAMS = 'USER_ENTRY_FILTER_MISSSING_MANDATORY_PARAMS;;One of the following parameters must be supplied: userIdEqual, userIdIn, entryIdEqual, entryIdIn ';
 	
```

Comment:
But using it the user will not know the rules that just on param of the following list is required.